starter-code/src/sequencer.js

```diff
@@ -1,68 +1,135 @@
-var LetterSequence = function(){}
-
-LetterSequence.prototype.createSequence = function(sequence){
-  var characters      = sequence.split("");
-  var containerString = "";
-  var repeatCount     = 1;
-
-  for (var i = 0; i < characters.length; i++){
-    var currentChar = characters[i];
-    var prevChar    = characters[i - 1];
-    var nextChar    = characters[i + 1];
+class LetterSequence {
+  constructor (){
+  }
 
-    if (currentChar === prevChar){
-      repeatCount++
+  createSequence (sequence) {
+    let characters      = sequence.split("");
+    let containerString = "";
+    let repeatCount     = 1;
+  
+    for (let i = 0; i < characters.length; i++){
+      let currentChar = characters[i];
+      let prevChar    = characters[i - 1];
+      let nextChar    = characters[i + 1];
+  
+      if (currentChar === prevChar){
+        repeatCount++
+      }
+  
+      // If the sequence is broken, and the repeat count is greater than 1
+      // add the letter and the repeat count to the return string
+      if (currentChar !== nextChar && repeatCount >= 1){
+        let repeats = repeatCount > 1 ? String(repeatCount) : ""
+        containerString += (repeats + currentChar)
+        repeatCount = 1;
+      }
     }
+  
+    return containerString;
+  }
 
-    // If the sequence is broken, and the repeat count is greater than 1
-    // add the letter and the repeat count to the return string
-    if (currentChar !== nextChar && repeatCount >= 1){
-      var repeats = repeatCount > 1 ? String(repeatCount) : ""
-      containerString += (repeats + currentChar)
-      repeatCount = 1;
-    }
+  _repeat (count, character){
```

Comment:
You don't have to define the `_repeat` method. You can use `String.prototype.repeat` instead.